tests/test_dna.py

```diff
@@ -1,5 +1,5 @@
 from hypothesis import given
-from minimal import minimal
+from .minimal import minimal
```

Comment:
Importing `minimal` this way means we can probably take it out of known third party (since it really isn't third party) from the pyproject.toml

Reply:
Ah. Didn't even realise we had it in as third party. That's confusing having a minimal.py which is the same name as a third party library